### PR TITLE
Update main.py and model.py to account for removal of VerificationMaterials from Sigstore

### PR DESCRIPTION
**Pull request by Andrew-Gan**
#### Summary

Sigstore 3.0 has removed VerificationMaterials as stated in [Release v3.0.0](https://github.com/sigstore/sigstore-python/releases/tag/v3.0.0)
Instead, verifier takes in bundles as argument now.

#### Release Note

Bug fix
Unable to run main.py according to README.md instructions using dependencies in install/requirements_Linux.txt.

#### Documentation

These changes enabled me to sign artifacts using the instructions on README.md

### CODE REVIEW

**Review by mihaimaruseac — COMMENTED** [review]
Please remove unrelated changes to main.py, none of those are needed.

Also, instead of commenting stale code, please remove it

**Review by mihaimaruseac — APPROVED** [review]
This LGTM, but leaving to @laurentsimon / @haydentherapper for final approval

**Comment by mihaimaruseac** [comment]
Actually, can you make your commits be signed, please? We use DCO, see https://github.com/sigstore/model-transparency/blob/main/CONTRIBUTING.md

**Review by mihaimaruseac — APPROVED** [review]

File: model_signing/model.py [hunk 1]
```diff
@@ -145,17 +142,13 @@ def verify(self, inputfn: Path, signaturefn: Path,
             bundle_bytes = signaturefn.read_bytes()
             bundle = Bundle().from_json(bundle_bytes)
 
-            material: tuple[Path, VerificationMaterials]
             contentio = io.BytesIO(Serializer.serialize_v1(
```
Comment (mihaimaruseac) [inline model_signing/model.py:145]: The `contentio` variable becomes unused now. Which means that we still don't serialize the model when we verify.

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Comment by laurentsimon** [comment]
@spencerschrock I think we missed the regression when updating our deps because we don't have e2e tests https://github.com/sigstore/model-transparency/issues/5. This issue is another good starting one

**Review by mihaimaruseac — DISMISSED** [review]

**Comment by mihaimaruseac** [comment]
Can you amend the last commit please to pass the DCO check? (same as https://github.com/sigstore/model-transparency/pull/212#issuecomment-2174639942)

**Review by mihaimaruseac — APPROVED** [review]
Thank you!

**Pull request merged**